components/movementsensor/imuwit/imu.go

```diff
@@ -153,7 +153,7 @@ func (imu *wit) Readings(ctx context.Context, extra map[string]interface{}) (map
 	}
 	readings["magnetometer"] = mag
 
-	acc, err := imu.GetAcceleration(ctx)
+	acc, err := imu.LinearAcceleration(ctx, extra)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Oh, shoot. I wish I'd thought to give this feedback earlier. For all sensors that support linear acceleration, we should modify their `Properties()` function (in this file, it's 7 lines below where I'm leaving this comment, because I can't comment that far from one of your changes directly) to indicate that they support linear acceleration. 

For sensors that don't support it, no changes to `Properties()` are needed.